sources/packages/backend/workflow/test/2021-2022/parttime-assessment/awards-amounts/parttime-assessment-awards-amount-CSGP.e2e-spec.ts

```diff
@@ -47,4 +47,26 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-CS
       0,
     );
   });
+
+  it("Should determine federalAwardCSGPAmount when awardEligibilityCSGP is true, no CSGP awarded in the program year previously and studentDataApplicationPDPPDStatus is true", async () => {
```

Comment:
It would be better if the test description reflects the real value.
Instead of
`and studentDataApplicationPDPPDStatus is true`
it would be
`and studentDataApplicationPDPPDStatus is 'yes'`.